packages/flutter/lib/src/material/chip.dart

```diff
@@ -1851,10 +1851,7 @@ class _RawChipState extends State<RawChip> with TickerProviderStateMixin<RawChip
           animation: Listenable.merge(<Listenable>[selectController, enableController]),
           builder: (BuildContext context, Widget child) {
             return Container(
-              decoration: ShapeDecoration(
-                shape: shape,
```

Comment:
My understanding is that the chip is already given its shape by the Material above (see line 1833) and this here is unnecessary.

(It in fact makes animating the shape of the chip more complicated)